package.json

```diff
@@ -44,7 +44,7 @@
     "lodash": "^4.17.11",
     "moment": "^2.22.2",
     "net": "^1.0.2",
-    "node-cardano-wallet": "https://github.com/Emurgo/node-cardano-wallet.git",
+    "node-cardano-wallet": "https://github.com/Emurgo/node-cardano-wallet.git#testnet-addr-checking",
```

Comment:
@rooooooooob @v-almonacid  maybe we should merge those ones and then update this PR with the final urls

Reply:
Yeah it mentions it. The `js-cardwano-wasm` PR just got merged by IOHK right now, so I updated the PRs in our `react-native-cardano` and `node-cardano-wallet` repos to refer to the IOHK main repo rather than my branch in my fork. Now we just need for those two to get finished before updating this PR.